common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -30,131 +30,134 @@ public class CommonValue {
 
     public static final String ML_MODEL_INDEX = ".plugins-ml-model";
     public static final String ML_TASK_INDEX = ".plugins-ml-task";
+    public static final String ML_MODEL_RELOAD_INDEX = ".plugins-ml-model-reload";
```

Comment:
Adding new index for a specific tasks is not extensible. Is it possible to reuse current index?